h/security/permissions.py

```diff
@@ -3,7 +3,7 @@
 
 class Permission:
     class Group(Enum):
-        ADMIN = "group:admin"  # Is this really "EDIT" or a combination?
```

Comment:
I think this was always intended to be the permission for "administrating" the group, and the group's creator is its "administrator". Conceptually it wasn't just about being able to "edit" the group (edit its name and description) but more generally being able to "administrate" it. In practice I'm not sure if it actually grants any other abilities though beyond editing the name and description. Maybe moderation capabilities?

Reply:
There's a note at top, but I think it's something like this. I think this was named after _who_ you are not _what_ you can do. So it's about the fact you are an admin (in this context) rather than because you can admin.

I suspect this then got confused and corrupted with all the other admin stuff as well. Before we started this work _all_ of these settings were just the string "admin" everywhere.

Moderation is actually a separate permission.